README.md

```diff
@@ -1 +1,2 @@
-# project
\ No newline at end of file
+# project
+akansha Bhardwaj
```

Comment:
change akanksha to akanksha1 and add new commit

Reply:
done